src/casp/services/db/models/ml_management.py

```diff
@@ -53,4 +38,4 @@ class CASMatchingEngineIndex(db.Base):
     def __str__(self):
         return self.index_name
 
-    __tablename__ = "cas_matching_engine_index"
+    __tablename__ = "ml_management_matchingengineindex"
```

Comment:
why no `_`s in `machineengineindex`?

Reply:
Please see the response to the previous conversation about that.